packages/utilities/src/useFocusRects.ts

```diff
@@ -40,33 +67,50 @@ type AppWindow = (Window & { FabricConfig?: { disableFocusRects?: boolean } }) |
  * @param rootRef - A Ref object. Focus rectangle can be applied on itself and all its children.
  */
 export function useFocusRects(rootRef?: React.RefObject<HTMLElement>): void {
+  const { providerRef } = React.useContext(FocusRectsContext);
+
   React.useEffect(() => {
     const win = getWindow(rootRef?.current) as AppWindow;
 
     if (!win || win.FabricConfig?.disableFocusRects === true) {
       return undefined;
     }
 
-    let count = setMountCounters(win, 1);
+    let el: Window | HTMLElement = win;
+    let onMouseDown: (ev: MouseEvent) => void;
+    let onPointerDown: (ev: PointerEvent) => void;
+    let onKeyUp: (ev: KeyboardEvent) => void;
+    if (providerRef && providerRef.current) {
+      el = providerRef.current;
+      const callbacks = setCallBackMap(el as HTMLElement);
```

Comment:
If I do that it gives me this error

![image](https://user-images.githubusercontent.com/7798177/180326103-e0c30591-1405-4c78-8ead-c81b92eeac8f.png)

Reply:
Full screenshot of the error. It freaks out because of the `useEffect` return callback using `el`:

![image](https://user-images.githubusercontent.com/7798177/180327279-19f6974a-e7c7-4f73-9f07-73846fe277b0.png)